addon/components/uni-carrousel.js

```diff
@@ -22,11 +21,11 @@ export default Component.extend(EKMixin, {
     return this.get('page') * this.get('componentWidth') * -1;
   }),
   wrapperStyle: computed('deltaX', function() {
-    return htmlSafe(`transform: translate3d(${this.get('deltaX')}px, 0px, 0)`);
+    return htmlSafe(`transform: translate3d(${parseFloat(this.deltaX)}px, 0px, 0)`);
```

Comment:
This does more than you know... you just made this only compatible with ember 3.0 onwards by moving from `this.get('deltaX')` to `this.deltaX`

Reply:
Addressed in the branch. Nice catch!